test/item_test.rb

```diff
@@ -1,5 +1,6 @@
 require_relative '../test/test_helper'
 require_relative '../lib/item'
+require_relative '../lib/sales_engine'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.